apps/docs/spec/supabase_js_v2.yml

```diff
@@ -2281,12 +2282,39 @@ functions:
             error: null
           }
           ```
+      - id: enroll-phone-factor
+        name: Enroll a Phone Factor
```

Comment:
In general for these examples, let's do two:

- One for enrolling (add new factor)
- One for verifying (after sign-in)